website/docs/release_notes.md

```diff
@@ -5,6 +5,140 @@ sidebar_position: 99
 
 # Release notes
 
+## [v0.1.15](https://github.com/VirtusLab/scala-cli/releases/tag/v0.1.15)
+
+## M1 native launcher is here! (experimental)
+
+We are happy to announce that there is new dedicated launcher for M1 users.
+
+Added in [#1396](https://github.com/VirtusLab/scala-cli/pull/1396) by [@lwronski](https://github.com/lwronski)
+
+## `--python` option for `repl` sub-command (experimental)
+
+Passing `--python` option allows using ScalaPy with `repl`:
+
+```
+▶ scala-cli --python
+Welcome to Scala 3.1.3 (17.0.2, Java OpenJDK 64-Bit Server VM).
+Type in expressions for evaluation. Or try :help.
+                                                                                
+scala> import me.shadaj.scalapy.py
+                                                                                
+scala> py.Dynamic.global.range(1, 4)
+val res0: me.shadaj.scalapy.py.Dynamic = range(1, 4)
+```
+
+Added in [#1336](https://github.com/VirtusLab/scala-cli/pull/1336) by [@alexarchambault](https://github.com/alexarchambault)
+
+## `-d`, `-classpath` and `compile` sub-command's `--output` options changes
+
+To be backward compatible with `scala` command a lot of changes have been made to the following options:
+* The `compile`'s  `--output` option has been renamed to `--compilation-output`. This option is now also available from the `run` and `package` sub-commands:
+
+```
+▶ scala-cli compile Hello.scala --compilation-output out
+▶ scala-cli --main-class Hello -classpath out
+Hello
+```
+
+* The `-d` option is no longer an alias for `--dependency`, but for `--compilation-output`. 
+  * `-O -d -O path/to/compilation/output` now defaults to `-d path/to/compilation/output`.
+
+```
+▶ scala-cli compile Hello.scala -d out
+▶ scala-cli --main-class Hello -classpath out
+Hello
+```
+
+* The old `--classpath` option has been renamed to `--print-classpath`.
+  *  `--classpath`, `--class-path` and `-classpath` options are now aliases for the `--extra jars` option.
+  * `-O -classpath -O path/to/classpath` now defaults to `--extra-jars path/to/classpath`.
+
+```
+▶ scala-cli compile --print-classpath Hello.scala
+# /work/.scala/project-cef76d561e/classes:~/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/scala-lang/scala-library/2.12.14/scala-library-2.12.14.jar:~/Library/Caches/ScalaCli/local-repo/0.1.0/org.virtuslab.scala-cli/runner_2.12/0.0.1-SNAPSHOT/jars/runner_2.12.jar:~/Library/Caches/ScalaCli/local-repo/0.1.0/org.virtuslab.scala-cli/stubs/0.0.1-SNAPSHOT/jars/stubs.jar
```

Comment:
it'd be better to show an example class path with an up-to date binary of `0.1.15`. (instead of a local dev output with `0.0.1-SNAPSHOT`)
you can produce a mock one locally, or just run `scala-cli compile Hello.scala --classpath` with `0.1.14` and change the versions in the output manually.